src/testing/jest/jest-27-and-under/matchers/events.ts

```diff
@@ -91,6 +91,33 @@ export function toHaveFirstReceivedEventDetail(eventSpy: d.EventSpy, eventDetail
 
   const pass = deepEqual(eventSpy.firstEvent.detail, eventDetail);
 
+  expect(eventSpy.firstEvent.detail).toEqual(eventDetail);
```

Comment:
Don't like how Git is showing the diff, but this line used to be: `expect(eventSpy.lastEvent.detail).toEqual(eventDetail)`

Changed it to use `firstEvent`, since, well, this is supposed to be checking what event data the spy first received. Idk why this was `lastEvent` before, but, in testing, I setup a component to emit multiple events and checked that the assertions using `toHaveFirstReceivedEventDetail` and `toHaveLastReceivedEventDetail` were working as expected with the emitted values.

I'm not sure if this is a breaking change, but seems like the original implementation was wrong in the first place 😅